packages/mds-cache/index.ts

```diff
@@ -317,10 +317,27 @@ async function writeDevice(device: Device) {
   return hwrite('device', device)
 }
 
+async function readKeys(pattern: string) {
+  return (await getClient()).keysAsync(pattern)
+}
+
+async function wipeDevice(device_id: UUID) {
+  const keys = await readKeys(`*:${device_id}:*`)
```

Comment:
I've been trained to NEVER use wildcards in Redis.  Probably should synthesize the three possibilities manually.